terra-jupyter-aou/Dockerfile

```diff
@@ -110,6 +110,15 @@ RUN curl -SL --output dotnet.tar.gz "https://dotnetcli.azureedge.net/dotnet/Runt
 
 RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 
+RUN wget https://github.com/samtools/bcftools/releases/download/1.12/bcftools-1.12.tar.bz2 && \
```

Comment:
No strong preference really, but wondering if this should be done from a tmp directory like `/tmp/bcftools`?  (Or maybe `/usr/local/share`?)

Reply:
+1 (see other tmpdir handling above for examples)

Reply:
sounds good